src/core/api/LoginProvider.ts

```
@@ -0,0 +1,16 @@
+
+interface LoginPayload {
+    email: string;
+    password: string;
+}
+
+class LoginProvider {
+    // La méthode login retourne la promesse de l'appel API
+    static login(payload: LoginPayload) {
+        return api.post("/login", payload);
+    }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)